src/Components/Artwork/__tests__/Details.jest.tsx

```diff
@@ -36,7 +36,7 @@ describe("Details", () => {
   ) => {
     return await renderRelayTree({
       Component: props => (
-        <ArtworkGridContextProvider isAuctionArtwork>
+        <ArtworkGridContextProvider isAuctionArtwork saveOnlyToDefaultList>
```

Comment:
Needed this prop explicitly in tests, now that the early return [here](https://github.com/artsy/force/pull/12938/files#diff-9e4884fa02e562601553700c318d04eff9f3e5b962c609aa5b68bae462472850L232) no longer kicks in.